### PR TITLE
Fixes #28186 - fix usability with custom audit actions

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
[test foreman]

**Comment by tbrisker** [comment]
@kgaikwad can you please take a look?

File: app/helpers/audits_helper.rb [hunk 1]
```diff
@@ -99,7 +99,7 @@ def audit_login?(audit)
   end
 
   def audit_action_name(audit)
-    return (audit.action == 'destroy') ? 'deleted' : "#{audit.action}d" if main_object? audit
+    return audit.action if main_object? audit
 
     case audit.action
```
Comment (kgaikwad) [inline app/helpers/audits_helper.rb:113]: For main objects, you have removed the code which was added to form a past tense of the action.
To make it consistent, could you please modify the behavior for audits that are not belongs to main objects?

**Review by kgaikwad — CHANGES_REQUESTED** [review]
Thank you @jlsherrill.

Added a small inline comment.  
Would it be possible to share any custom action example so that I can test changes against it as well.

File: app/helpers/audits_helper.rb (hunk 1 above)
Comment (jlsherrill) [reply app/helpers/audits_helper.rb:113]: updated!

**Comment by jlsherrill** [comment]
There's an example of a custom audit action here: https://github.com/Katello/katello/pull/8409

**Comment by jlsherrill** [comment]
[test foreman]

File: app/helpers/audits_helper.rb (hunk 1 above)
Comment (kgaikwad) [reply app/helpers/audits_helper.rb:113]: @jlsherrill, thank you for updates.
There are few places where proper action names handling needs to be done. 
With this PR changes, for add/remove actions it shows full attributes old/new changes but it is expected to show only the description that object is added or removed from main object. 
For example - 
Screenshot from 2019-11-15 18-13-32

1. In audit_helper at [line 85](https://github.com/theforeman/foreman/blob/develop/app/helpers/audits_helper.rb#L85) and [line 231](https://github.com/theforeman/foreman/blob/develop/app/helpers/audits_helper.rb#L231) 
2. Inside react component i.e AuditsList > ExpansiveView, at [line 51](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/AuditsList/ExpansiveView.js#L51)

Comment (jlsherrill) [reply app/helpers/audits_helper.rb:113]: Thanks! Good catches!  I updated line 85 quite a bit to be i18n'd.

**Review by kgaikwad — APPROVED** [review]
ACK from me. Thank you @jlsherrill 

@tbrisker,
I think, I don't have enough rights to perform pull-request merge. 
Do you have any more suggestions?

**Review by tbrisker — APPROVED** [review]
Thanks @jlsherrill and @kgaikwad !

**Pull request merged**